### DISCUSSION TITLE
Git revision date and authors

### DISCUSSION THREAD

**Discussion by kajulnisha9721** [squidfunk/mkdocs-material#5796] [Q&A]
How to display the git summary that contains **git revision date and authors in same line** as same as the material site,
My case I'm using **bitbucket**, so installed plugin of **git-authors**

### plugins      
plugins:
  - git-authors:
        show_contribution: true
        show_line_count: true
        show_email_address: true
        count_empty_lines: true
        fallback_to_empty: true
        sort_authors_by: name
        authorship_threshold_percent: 10
        exclude:
            - index.md
        enabled: true
        strict: true
  - git-revision-date-localized:
      type: timeago
      custom_format: "%d. %B %Y"
      timezone: Europe/Amsterdam
      locale: en
      fallback_to_build_date: true
      enable_creation_date: true
      exclude:
          - index.md
      enabled: true
      strict: true  
      
    ## my output
![image](https://github.com/squidfunk/mkdocs-material/assets/83535981/8476e8c7-e9ca-497e-a010-e772e532b761)

But I want it to be,
![image](https://github.com/squidfunk/mkdocs-material/assets/83535981/642e40a8-de6e-4a97-94a0-a6981b76663c)

How to do that? i didn't find any examples on this. please help me!

**Comment by squidfunk** [comment]
The new layout is currently reserved to sponsors of the project and solely available via [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/) 😊

**Reply by kajulnisha9721** [reply]
Thanks for the reply, 

In the below snap, can you see the analytics overwritten with the git repo information, as it renders author below!
![image](https://github.com/squidfunk/mkdocs-material/assets/83535981/db2617f0-fe98-4724-9cf1-cc2b3c1f3e9c)

**overrides/main.html**
  ``` 
{% block content %}
  {{ super() }}
{% if git_page_authors %}
    <div class="md-source-date">
      <small>
        Author(s): {{ git_page_authors | default('enable mkdocs-git-authors-plugin') }}
      </small>
    </div>
  {% endif %}
{% endblock %}
```


**Why this is happening? is this a bug?**

**Reply by squidfunk** [reply]
The bug could be in your overrides or in our code. If you remove your overrides and the issue persists, please create a minimal reproduction and file a bug report. If it's in your code, you can ask other users for help 😊
